packages/components/src/circular-progress-bar/index.tsx

```diff
@@ -1,21 +1,28 @@
+import classnames from 'classnames';
 import './style.scss';
 
 const CircularProgressBar = ( {
 	currentStep,
 	numberOfSteps,
+	size,
```

Comment:
Is there any value in making this optional with a default value?

Reply:
We could make `size` optional, but I also wanted to ensure that developers always set a size for this component.